sharding/collation_test.go

```diff
@@ -43,6 +42,101 @@ func TestCollation_AddTransactions(t *testing.T) {
 	}
 }
 
+//TODO: Add test for converting *types.Transaction into raw blobs
```

Comment:
Wanna create an issue for this TODO? That way anyone can pick it up

Reply:
Sure thing , will create it once it is merged in